packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -108,6 +108,8 @@ class FrameFlameChart extends CoreElement {
 
     drawCpuEvents();
 
+    // TODO(kenzie): improve this by adding a spacer div instead of just
+    //  increasing the row. Do this once each section is in its own container.
```

Comment:
Nit: extra space before `increasing`.

Reply:
Ah that's IntelliJ trying to make it a multiline comment. Will remove.